test/integration/authrep/basic_test.rb

```diff
@@ -246,6 +246,56 @@ def setup
 
       assert_not_authorized
       assertions.call false
+
+      # Test that hitting children over the parent limit does not translate to
```

Comment:
Do you mean checking the XML response or checking later on that the values were actually added properly to the database? If the latter, this is already done via the sequence of tests, which have dependencies, ie. we add enough hits to child1 so that in a later call just adding 1 hit would mean it does not authorize, and we add enough hits initially to child2 so that the following authrep could not be authorised if it did add the hits to its parents.

Reply:
Maybe checking the XML is too much. That part should be well covered by other tests.
:+1: